Sources/Testing/Traits/Trait.swift

```diff
@@ -136,43 +140,46 @@ public protocol TestScoping: Sendable {
   /// this method on all non-`nil` scope providers, giving each an opportunity
   /// to perform arbitrary work before or after invoking `function`.
   ///
-  /// This method should either invoke `function` once before returning or throw
-  /// an error if it is unable to provide a custom scope.
+  /// This method should either invoke `function` once before returning,
+  /// or throw an error if it's unable to provide a custom scope.
   ///
   /// Issues recorded by this method are associated with `test`.
   func provideScope(for test: Test, testCase: Test.Case?, performing function: @Sendable () async throws -> Void) async throws
 }
 
 extension Trait where Self: TestScoping {
-  /// Get this trait's scope provider for the specified test and/or test case,
-  /// if any.
+  /// Get this trait's scope provider for the specified test or test case.
   ///
   /// - Parameters:
-  ///   - test: The test for which a scope provider is being requested.
-  ///   - testCase: The test case for which a scope provider is being requested,
-  ///     if any. When `test` represents a suite, the value of this argument is
+  ///   - test: The test for which the testing library requests a
+  ///     scope provider.
+  ///   - testCase: The test case for which the testing library requests a scope
+  ///     provider, if any. When `test` represents a suite, the value of this argument is
   ///     `nil`.
   ///
-  /// This default implementation is used when this trait type conforms to
-  /// ``TestScoping`` and its return value is discussed in
+  /// The testing library uses the default implementation of this method when
```

Comment:
We specifically use _this_ implementation when those constraints are met. How about:

```swift
/// The testing library uses this implementation of
/// ``Trait/scopeProvider(for:testCase:)-cjmg`` when the trait type conforms to
/// ``TestScoping``.
```

And drop "For information this [...]" since we'll have added a link?